jerry-ext/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+# Copyright JS Foundation and other contributors, http://js.foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cmake_minimum_required (VERSION 2.8.12)
+set(JERRY_EXT_NAME jerry-ext)
+project (${JERRY_EXT_NAME} C)
+
+# Include directories
+set(INCLUDE_EXT "${CMAKE_CURRENT_SOURCE_DIR}/include")
+
+# Source directories
+file(GLOB SOURCE_EXT args/*.c)
+
+add_library(${JERRY_EXT_NAME} STATIC ${SOURCE_EXT})
+
+target_include_directories(${JERRY_EXT_NAME} PUBLIC ${INCLUDE_EXT})
+target_include_directories(${JERRY_EXT_NAME} PRIVATE "${CMAKE_SOURCE_DIR}/jerry-core")
+target_link_libraries(${JERRY_NAME} jerry-core)
```

Comment:
Oh, I didn't found that typo! Thanks!

For the `target_link_libraries(${TARGET_NAME} jerry-core)` in unittest cmake, yes, we can remove it, but I prefer keeping it. Based on my understanding, the project (unit-ext) use the interface of jerry-core and jerry-ext, so it's natural to link both 2 library. Though those 2 libs have some internal dependency relations, but the user (unit-ext) doesn't know that.

What do you think?